Dockerfile.train

```diff
@@ -81,6 +93,9 @@ RUN LATEST_STABLE_RELEASE=$(curl "https://api.github.com/repos/coqui-ai/STT/rele
 # and we don't want to break that
 RUN DS_NODECODER=y DS_NOTENSORFLOW=y pip install --upgrade -e .
 
+# Install coqui_stt_training (inside tf-venv) for exporting models using tflite
+RUN /tflite-venv/bin/pip install coqui_stt_training torch
```

Comment:
`torch`?

Reply:
@reuben This is the error if you try without installing torch. (Had to use `PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python` to workaround protobuff changes)

```
root@0f60877c1814:/code# PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python /tflite-venv/bin/python -m coqui_stt_training.export --alphabet_config_path /mnt/models/alphabet.txt --scorer_path /mnt/lm/kenlm.scorer --feature_cache /mnt/sources/feature_cache --n_hidden 2048 --beam_width 500 --lm_alpha 1.6383103526118539 --lm_beta 0.1291025653672666 --load_evaluate best --checkpoint_dir /mnt/checkpoints/ --export_dir /mnt/models/ --export_tflite true --export_author_id CommonVoice-FR-Team --export_model_version 0.9 --export_contact_info https://discourse.mozilla.org/c/voice/fr --export_license MIT-0 --export_language fr-FR --export_min_stt_version 1.0.0 --export_max_stt_version 1.4.0 --export_model_name cv-fr-tflitecoqui_stt_training.export --alphabet_config_path /mnt/models/alphabet.txt --scorer_path /mnt/lm/kenlm.scorer --feature_cache /mnt/sources/feature_cache --n_hidden 2048 --beam_width 500 --lm_alpha 1.6383103526118539 --lm_beta 0.1291025653672666 --load_evaluate best --checkpoint_dir /mnt/checkpoints/ --export_dir /mnt/models/ --export_tflite true --export_author_id CommonVoice-FR-Team --export_model_version 0.9 --export_contact_info https://discourse.mozilla.org/c/voice/fr --export_license MIT-0 --export_language fr-FR --export_min_stt_version 1.0.0 --export_max_stt_version 1.4.0 --export_model_name cv-fr-tflite
Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tflite-venv/lib/python3.7/site-packages/coqui_stt_training/export.py", line 17, in <module>
    from .deepspeech_model import create_inference_graph, create_model, reset_default_graph
  File "/tflite-venv/lib/python3.7/site-packages/coqui_stt_training/deepspeech_model.py", line 25, in <module>
    from .util.config import Config
  File "/tflite-venv/lib/python3.7/site-packages/coqui_stt_training/util/config.py", line 16, in <module>
    from .augmentations import NormalizeSampleRate, parse_augmentations
  File "/tflite-venv/lib/python3.7/site-packages/coqui_stt_training/util/augmentations.py", line 26, in <module>
    from .sample_collections import samples_from_source, unpack_maybe
  File "/tflite-venv/lib/python3.7/site-packages/coqui_stt_training/util/sample_collections.py", line 21, in <module>
    import webdataset as wds
  File "/tflite-venv/lib/python3.7/site-packages/webdataset/__init__.py", line 72, in <module>
    from .mix import RandomMix, RoundRobin
  File "/tflite-venv/lib/python3.7/site-packages/webdataset/mix.py", line 16, in <module>
    from torch.utils.data import IterableDataset
ModuleNotFoundError: No module named 'torch'
```

Reply:
We come back to `webdataset` unregistered dependency for `torch`. On my image for cv-fr I need to install it twice... once for the system (py38) and once in the venv (py37).... so annoying.

Reply:
We can install from source and pin torch from setup.py 📌